app/src/pages/homePage/HomePage.test.tsx

```diff
@@ -28,29 +29,42 @@ describe('StartPage', () => {
         mockNavigate.mockImplementation(() => mockUseNavigate);
 
         const contentStrings = [
-            'This service gives you access to Lloyd George digital health records.',
-            'You can use this service if you are:',
-            'part of a GP practise and need to view, download or remove a patient record',
-            'managing records on behalf of NHS England and need to download a patient record',
+            'This service gives you access to Lloyd George digital health records. ' +
+                'You may have received a note within a patient record, stating that the record has been digitised.',
+            'If you are part of a GP practice, you can use this service to:',
+            'view a patient record',
+            'remove a patient record',
+            'If you are managing records on behalf of NHS England, you can:',
             'Not every patient will have a digital record available.',
-            'Before You Start',
-            "You'll be asked for:",
+            'Before you start',
+            'You’ll be asked for:',
             'your NHS smartcard',
             'patient details including their name, date of birth and NHS number',
         ];
 
+        const contentStringsAppearMoreThanOnce = {
+            'download a patient record': 2,
+        };
+
         render(<HomePage />);
+
         contentStrings.forEach((s) => {
             expect(screen.getByText(s)).toBeInTheDocument();
         });
+        Object.entries(contentStringsAppearMoreThanOnce).forEach(([s, count]) => {
+            const foundElements = screen.getAllByText(s);
+            expect(foundElements).toHaveLength(count);
+            foundElements.forEach((element) => expect(element).toBeInTheDocument());
+        });
+
```

Comment:
This feels like over-engineering to check for multiple instances of a string only once. Maybe if there was a a bigger map.

` 
const foundElements = screen.getAllByText('download a patient record');
expect(foundElements).toHaveLength(2);
`
This should be fine

Reply:
Agree. Probably we won't have another duplicated string on this page. Thanks and suggestion adopted :)